packages/discovery-provider/ddl/migrations/0083_create_comments.sql

```diff
@@ -0,0 +1,29 @@
+begin;
+
+CREATE TABLE IF NOT EXISTS comments (
+    comment_id INTEGER PRIMARY KEY,
+    text TEXT NOT NULL,
+    user_id INTEGER NOT NULL,
+    entity_id INTEGER NOT NULL,
+    entity_type TEXT NOT NULL,
+    track_timestamp_ms BIGINT,
```

Comment:
I don't think this needs to be a big int...

32 bit integers can be up to 4,294,967,295 unsigned or 2,147,483,647 signed

2,147,483,647 / 1000 ms/s / 60 s/min / 60 min/hr =~ 596 hrs, I don't think we have any plans to support that length!